### PR TITLE
Download and upload handout file on MongoDB

### PR DESCRIPTION
**Pull request by sendr**

### CODE REVIEW

File: video_xblock/video_xblock.py [hunk 1]
```diff
@@ -81,7 +81,14 @@ class VideoXBlock(StudioEditableXBlockMixin, XBlock):
         help="Video muted or not"
     )
 
-    editable_fields = ('display_name', 'href', 'account_id')
+    handout = String(
+        default='',
+        scope=Scope.content,
+        display_name='Upload handout',
+        help='You can upload handout file for students'
```
Comment (polesye) [inline video_xblock/video_xblock.py]: please add i18n

File: video_xblock/video_xblock.py [hunk 2]
```diff
@@ -209,3 +245,19 @@ def get_player(self):
         """
         player = BaseVideoPlayer.load_class(self.player_name)
         return player()
+
+    def _make_field_info(self, field_name, field):
```
Comment (polesye) [inline video_xblock/video_xblock.py]: add docstring

**Comment by polesye** [comment]
@xahgmah or @dorosh  please also review.
@z4y4ts just FYI

File: video_xblock/video_xblock.py (hunk 1 above)
Comment (sendr) [inline video_xblock/video_xblock.py]: done

**Comment by polesye** [comment]
don't forget to squash before merge

**Pull request merged**